asset-services/asset-services-api/src/errors.rs

```diff
@@ -1,25 +1,28 @@
 //! Error representation helpers.
 
-use asset_services_celery::exports::celery_prelude::CeleryError;
 use axum::http::{Response, StatusCode};
 use axum::response::IntoResponse;
 
-/// Map Celery errors to Internal Server Error responses.
-pub struct CeleryErrorResponse(CeleryError);
+/// Represent arbitrary errors as Axum Internal Server Error responses using [`anyhow`].
+#[repr(transparent)]
+pub struct AnyhowErrorResponse(anyhow::Error);
```

Comment:
I am not 100% sure what the need is for specifying `repr(transparent)` here. Are we transmuting between the wrapping and inner type, or is this to get the `.context()` methods to work with the wrapping type?

Reply:
It's not required, but I like specifying it as a form of executable documentation that the struct is intended as a newtype wrapper, to remove ambiguity (analogous to Haskell's distinction between the `newtype` and `data` keywords).

The main practical upshot is that if anything changes that stops it from being a newtype, then this will raise a compiler error for investigation rather than silently proceeding.